subjects/regressionTests.py

```diff
@@ -82,17 +82,23 @@ def cmdline_split(s):
         a.append(sub)
     return a
 
-# On Windows file removal is asynchronous but renames are atomic
-def remove_file(file):
-    os.rename(file, '%s.rmtmp' % file)
-    os.remove('%s.rmtmp' % file)
-
 # Remove output files
 def clear_dir(dir_name, files):
+    failedFiles = []
     for pname in files:
         for ext in output_extensions:
             if pname.endswith(ext):
-                remove_file(os.path.join(dir_name, pname))
+                filename = os.path.join(dir_name, pname)
+                try:
+                    os.remove(filename)
+                except:
+                    failedFiles.append(filename)
```

Comment:
Let's add a comment here like
`# Malware scanners may block the deletion, so try again later.`